SPECS/core-packages/core-packages.spec

```diff
@@ -73,6 +73,8 @@ Requires:       rpm
 Requires:       rpm-libs
 Requires:       sed
 Requires:       sqlite-libs
+Requires:       SymCrypt
+Requires:       SymCrypt-openssl
```

Comment:
Would it be better to have `SymCrypt-openssl` use `Requires: SymCrypt`? Or is the dependency relationship more complex?

Reply:
Right now `SymCrypt-openssl` has a `BuildRequires: SymCrypt`. I'd rather leave it that way, and I prefer the very explicit dependency here.